apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect, BrowserContext } from '@playwright/test'
+import { urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+
+const homeUrl = `${urls.islandisBaseUrl}/minarsidur`
+const sessionHistoryUrl = '/minarsidur/adgangsstyring/notkun'
+
+test.use({ baseURL: urls.islandisBaseUrl })
+
+test.describe('Service portal, in session history', () => {
+  let context: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    context = await session({
+      browser: browser,
+      storageState: 'service-portal-faereyjar.json',
+      homeUrl,
+      phoneNumber: '0102399',
+      idsLoginOn: true,
+    })
+  })
+
+  test.afterAll(async () => {
+    await context.close()
+  })
+
+  test('can view list of sessions', async () => {
+    // Arrange
+    const page = await context.newPage()
+
+    // Act
+    const sessionsRows = page.locator('table > tbody > tr')
+    await page.goto(sessionHistoryUrl, {
+      waitUntil: 'networkidle',
+    })
+
+    // Assert
+    await expect(sessionsRows).toHaveCountGreaterThan(0)
```

Comment:
I'm expecting sessionRows to be detached on `goto`

Reply:
Yes think a reorder between arrange and act confused the order. But for some strange reasons this actually works. But I reversed the order to be more logical.